lib/osf-components/addon/components/collection-submission-card/template.hbs

```diff
@@ -0,0 +1,48 @@
+<div local-class='page-container'>
+    <div local-class='icon-container'>
+        <FaIcon @icon='clock' @prefix='far' />
+    </div>
+    <div local-class='text-container'>
+        <div local-class='project-title'>
+            {{t 'osf-components.moderators.all.button'}}
+        </div>
+        <div local-class='moderation-details'>
+            {{t
+                'osf-components.moderators.all.moderationDetails'
+                status=@collectionSubmission.status
+                date=@date
+                moderator=@moderator
+            }}
+        </div>
+    </div>
+    <div local-class='button-container'>
+        <Button
+            @type='create'
+            {{on 'click' this.openMakeDecisionDialog}}
+            data-test-moderator-make-decision-button
+        >
+            {{t 'osf-components.moderators.all.button'}}
+            <FaIcon @icon='caret-down' />
+        </Button>
+    </div>
+    <OsfDialog
+        @isOpen={{this.shouldOpenDecisionDialog}}
+        @onClose={{this.closeDecisionDialog}}
+        @closeOnOutsideClick={{true}}
+        as |dialog|
+    >
```

Comment:
Same here, the dropdown should be taken care of by the `<MakeDecisionDropdown>` being updated in #1693 but this will be a good placeholder until that work is done.